pkg/domain/infra/abi/containers.go

```diff
@@ -834,6 +834,11 @@ func (ic *ContainerEngine) ContainerRun(ctx context.Context, opts entities.Conta
 	for _, w := range warn {
 		fmt.Fprintf(os.Stderr, "%s\n", w)
 	}
+
+	if opts.Rm {
```

Comment:
Probably better to put this in cmd/podman/common/specgen.go - that way it's not just `podman run` for the ABI implementation, but run and create for both local and remote

Reply:
thanks for the hint.  Fixed now